api.planx.uk/admin/migrations/splitFindProperty.ts

```diff
@@ -0,0 +1,165 @@
+import { Request, Response, NextFunction } from 'express';
+import { gql } from "graphql-request";
+import { customAlphabet } from "nanoid-good";
+import en from "nanoid-good/locale/en";
+
+import { adminGraphQLClient as client } from "../../hasura";
+import { Flow, PublishedFlow } from "../../types";
+
+/**
+ * This migration ran in February 2023
+ * 
+ * Context: FindProperty is split from two pages into two separate components (FindProperty + PlanningInformation) to faciliate overriding the "property.type"
+ * 
+ * What this does: Inserts a PlanningInformation component with default props (override = false) directly after a FindProperty in a flow schema. Additionally updates the currently published version of that flow
+ * 
+ * Considerations:
+ *   - nodeIds for the new PlanningInformation must be unique among flows to prevent clone behavior if used as portals
+ *   - a flow and its' currently published version need to share a nodeId so that it's not picked up in the publishing diff
+ *   - we assume one FindProperty per flow
+ *   - we assume no flow has PropertyInformation yet
+ *   - we ignore external portal nodes when scanning a flow schema (external portals are flows and can be migrated independently)
+ *   - we do not insert a corresponding operation, as "migrations" don't have actors, shouldn't need to be rolled-back, etc
+ * 
+ */
```

Comment:
since migrations are essentially one-off and fairly bespoke jobs, I figured I'd leave context & assumptions directly in the file itself. open to suggestions for this if other documentation preferences!